src/Microsoft.IdentityModel.Tokens/InMemoryCryptoProviderCache.cs

```diff
@@ -79,7 +79,7 @@ internal InMemoryCryptoProviderCache(CryptoProviderCacheOptions cryptoProviderCa
                 throw LogHelper.LogArgumentNullException(nameof(cryptoProviderCacheOptions));
 
             if (tryTakeTimeout <= 0)
-                throw LogHelper.LogArgumentException<ArgumentException>(nameof(tryTakeTimeout), $"{nameof(tryTakeTimeout)} must be greater than zero");
+                throw LogHelper.LogArgumentException<ArgumentException>(nameof(tryTakeTimeout), $"{LogHelper.MarkAsNonPII(nameof(tryTakeTimeout))} must be greater than zero");
```

Comment:
the second param here is a string right? Does the MarkAsNonPII do anything here?

Reply:
You're right. Its a string and marking it as NonPII would just create another object.

Reply:
Updated